### ISSUE TITLE
General Issue: Add updating `patch` to quarterly release process

### ISSUE THREAD

**Issue by bundfussr** [pharmaverse/admiraldev#199]
### Background Information

At the end of a quarterly release `main` should be merged to `patch`. Then the branch is up to date and all the changes of the release do not occur in the first pull request to `patch` (see for example https://github.com/pharmaverse/admiral/pull/1642).

**Comment by bms63** [comment]
I think we were looking into this from a CI perspective. I was thinking two PRs could auto-generate when pre-release is merged to main - one for patch and for devel. @cicdguy is this a good idea??  Be fun to see if I can figure this out with your blessing

**Comment by bms63** [comment]
## Scenario 1
When we have a critical bug that is identified form a user or from CRAN and we do a `patch > pre-release > main` to address the issue and re-publish to CRAN - so  `X.Y.1`, `X.Y.2`, `X.Y.#` release, etc.  A lot of the times we forget to merge the `patch` back into `devel`.  I was thinking that an automatic PR would be created once a `patch` branch has been merged into the `main`.  So this PR would merge changes in `main` back into `devel` when a `X.Y.#` is released and auto-assign people to the PR review.

## Scenario 2
The `patch` branch can quickly get out of sync with the `main` branch once all the development work is done for a major/minor release `#.#.Z`  When a new major/minor release is deployed then an automatic PR is created that will merge `main` into `patch` with auto-assigning of reviewers.

Since we are early in the release process again I am very eager to review our branching process again.  Any advice is very appreciated and we can re-imagine our branching strategy on best practices.   Any other thoughts admiral team @pharmaverse/admiral ?

**Comment by bms63** [comment]
A blurb should go in here.  

![Image](https://user-images.githubusercontent.com/10111024/221601604-05036bfa-9235-433c-aef8-acf32718cfbe.png)
